packages/scripts/utils/config.js

```diff
@@ -24,9 +25,35 @@ const hasBabelConfig = () =>
 	hasProjectFile( '.babelrc' ) ||
 	hasPackageProp( 'babel' );
 
+/**
+ * Returns path to a Jest configuration which should be provided as the explicit
+ * configuration when there is none available for discovery by Jest in the
+ * project environment. Returns undefined if Jest should be allowed to discover
+ * an available configuration.
+ *
+ * This can be used in cases where multiple possible configurations are
+ * supported. Since Jest will only discover `jest.config.js`, or `jest` package
+ * directive, such custom configurations must be specified explicitly.
+ *
+ * @param {"e2e"|"unit"} suffix Suffix of configuration file to accept.
+ *
+ * @return {string=} Override or fallback configuration file path.
+ */
+const getJestOverrideConfigFile = ( suffix ) =>
+	cond( [
```

Comment:
> I personally find it harder to follow when using `cond` rather than the series of `if` statements. I'm sure it's caused by seeing this helper in action in the project for the first time.

Yeah, I definitely struggled with trying to find a nice balance between readability and expressiveness (especially considering the precedent set by other functions of this file). I'll try to tinker with it a bit longer to see if I can come up with something better, else I'll see if simple `if` statements will suffice.

Reply:
`if`s are much simpler 😅 

See 15ff2c3

Reply:
Indeed, however it is quite hard to express it in the order that guarantees that it becomes so easy to follow as it is now. 👍